### PR TITLE
Added TypeScript Native ver 0.0.35-alpha

### PR DESCRIPTION
**Pull request by ASDAlexander77**
Added TypeScript Native ver 0.0.35-alpha

### CODE REVIEW

File: lib/compilers/typescript-native.ts [hunk 1]
```diff
@@ -41,107 +41,83 @@ export class TypeScriptNativeCompiler extends BaseCompiler {
...
 
-    override async generateIR(
-        inputFilename: string,
-        options: string[],
-        irOptions: LLVMIrBackendOptions,
-        filters: ParseFiltersAndOutputOptions,
-    ) {
-        // These options make Clang produce an IR
-        let newOptions = ['--emit=llvm', inputFilename];
+    override async generateIR(inputFilename: string, options: string[], filters: ParseFiltersAndOutputOptions) {
```
Comment (jeremy-rifkin) [inline lib/compilers/typescript-native.ts:96]: This new function signature seems like a mistake, and I'm surprised it compiles. The third argument isn't of type `ParseFiltersAndOutputOptions` in the base compiler.

Comment (ASDAlexander77) [reply lib/compilers/typescript-native.ts]: updated, can u review it again, pls

**Review by mattgodbolt — APPROVED** [review]
Seems good to me now, thanks.

**Comment by mattgodbolt** [comment]
This is now live: thanks!

**Pull request merged**